recipes/libsndfile/all/conandata.yml

```diff
@@ -1,11 +1,17 @@
 sources:
-  "1.0.29":
-    url: "https://github.com/erikd/libsndfile/archive/v1.0.29.tar.gz"
-    sha256: "c5541ccfa672b3f461ac21a38cdd37b1fe061cc62a422800b9d53649dee549ee"
+  "1.0.31":
+    url: "https://github.com/libsndfile/libsndfile/releases/download/1.0.31/libsndfile-1.0.31.tar.bz2"
```

Comment:
For linux is totally okay, but is bz2 well supported on Windows and Mac?

Reply:
The official python binaries have bz2 support.

Reply:
Great! so we good to go, thank you!